app/models/validations/property_validations.rb

```diff
@@ -35,8 +37,10 @@ def validate_unitletas(record)
   end
 
   def validate_property_postcode(record)
-    if record.postcode_known == "Yes" && record.property_postcode.blank?
-      record.errors.add :property_postcode, "Enter a postcode in the correct format, for example AA1 1AA"
+    postcode = record.property_postcode
+    if record.postcode_known == "Yes" && (postcode.blank? || !postcode.match(POSTCODE_REGEXP))
```

Comment:
I _think_ we don't want the blank check here so that users aren't blocked on this question? (i.e. they can skip and come back to it?)

Reply:
I don't think they are, at least this was working when I tested.

Reply:
The use case I'm thinking of is:

1. Do you know the postcode answered as "Yes" (I know I have it, I just don't have it in front of me right now)
2. Don't fill in the postcode just click save and continue (I'll go look it up in a minute when I've filled out the rest of the section)
3. Am I now blocked on postcode by this validation?

Reply:
I think this is the behaviour we expect as defined in CLDC-783

Reply:
Looks like this is agreed now so will mark this as resolved. Might be worth updating this https://github.com/communitiesuk/mhclg-data-collection-beta/blob/main/docs/adr/adr-007-data-validations.md to reflect that.